docs/articles/documentation/reference/test-api/requesthook/README.md

```diff
@@ -103,3 +103,35 @@ async onResponse (event) {
     throw new Error('Not implemented');
 }
 ```
+
+{% include notes-warnings/requesthook-onresponse-no-stub-headers.md href='#change-or-delete-response-headers' %}
+
+### Change or Delete Response Headers
+
+You can set or remove response headers in a `RequestHook._onConfigureResponse()` call. This allows you to modify part of the response headers before the response is sent to the browser.
+
+To change response headers, use the `event.setHeaders` and `event.removeHeader` methods in the `__onConfigureResponse` call.
+
+> Important! The `RequestHook._onConfigureResponse()` method is not part of TestCafe public API. As a private method, it can be removed or deprecated in the upcoming releases without prior warning. Be cautious when you add this method to your request hooks.
```

Comment:
```suggestion
> Important! The `RequestHook._onConfigureResponse()` method is not part of TestCafe public API. As a private method, it can be removed or deprecated in the upcoming releases without prior warning. Be cautious when you override this method in your request hooks.
```